src/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/FileSecurity.cs

```diff
@@ -615,35 +17,12 @@ public FileSecurity()
         public FileSecurity(string fileName, AccessControlSections includeSections)
             : base(false, fileName, includeSections, false)
         {
-            string fullPath = Path.GetFullPath(fileName);
+            Path.GetFullPath(fileName);
```

Comment:
What is the purpose of this? To verify if the path can be normalized? Because we are not passing it anywhere.

Reply:
It will still throw if you have an embedded null.